ci/benchmark/build.sh

```diff
@@ -37,10 +37,10 @@ export GBENCH_BENCHMARKS_DIR="$WORKSPACE/cpp/build/gbenchmarks/"
 export LIBCUDF_KERNEL_CACHE_PATH="$HOME/.jitify-cache"
 
 # Dask & Distributed option to install main(nightly) or `conda-forge` packages.
-export INSTALL_DASK_MAIN=0
+export INSTALL_DASK_MAIN=1
 
 # Dask version to install when `INSTALL_DASK_MAIN=0`
-export DASK_STABLE_VERSION="2022.11.1"
+export DASK_STABLE_VERSION="2022.12.0"
```

Comment:
Any reason why we're bumping the minimum version to `2022.12.0` in addition to unpinning?

Reply:
Because of the code-cleanup in `dask-cuda`: https://github.com/rapidsai/dask-cuda/pull/1060